client/src/components/App/index.js

```diff
@@ -173,6 +174,10 @@ const App = () => {
               path="/dashboard/applications/submitted/:applicantID"
               render={(props) => <SubmittedId {...props} />}
             />
+            <Route
+              path="/dashboard/applications/completed/:applicantID"
+              render={(props) => <SubmittedId {...props} />}
+            />
```

Comment:
you should `exact` to the link to avoid displaying two pages that has nearly similar endpoints Over each other, this is cause problems and repeat that for all routes in dashboard, Admin side. 
```suggestion
            <Route
               exact
              path="/dashboard/applications/completed/:applicantID"
              render={(props) => <SubmittedId {...props} />}
            />
```